packages/picasso/src/ScrollMenu/ScrollMenu.tsx

```diff
@@ -75,6 +75,10 @@ const ScrollMenu: FunctionComponent<Props> = props => {
       className={classes.menu}
       style={style}
       role={role}
+      onMouseDown={e => {
+        // ScrollMenu is used in dropdowns. Prevents blur --> dropdown close when scrolled.
+        e.preventDefault()
+      }}
```

Comment:
this is unlikely, but if someone passes his own `onMouseDown` implementation, the bug will return

Reply:
good catch! :) I might want to pass it right after the `rest`

Reply:
so, does it mean, that we are not allowing users to set `onMouseDown`, is it intentional?
```suggestion
     onMouseDown={e => {
        // ScrollMenu is used in dropdowns. Prevents blur --> dropdown close when scrolled.
        e.preventDefault()
        
        if (typeof rest.onMouseDown === 'function) {
          rest.onMouseDown(e)
        }
      }}
```

How about this?

Reply:
This is also an option. I thought we don't need it as the component is internal. But I like this variant.

Reply:
Checked it again. Looks like the current interface of a component does not really have `onMouseDown`. So TS does not allow to pass it and it is kinda not expected to have it. I don't see the reason to add it currently.